.github/workflows/job_pytorch_layer_tests.yml

```diff
@@ -0,0 +1,167 @@
+name: PyTorch Layer Tests
+
+on:
+  workflow_call:
+    inputs:
+      runner:
+        description: 'Machine on which the tests would run'
+        type: string
+        required: true
+      shell:
+        description: "shell to override the default shell settings in the runner's operating system."
+        type: string
+        required: true
```

Comment:
It could be done directly in the `shell` of the job definition like in https://github.com/openvinotoolkit/openvino/blob/2507d89d9cfcf38bf43238aaf83ff74f066fe452/.github/workflows/job_cxx_unit_tests.yml#L31

Reply:
let me fix in separate PR.